apps/passport-server/src/database/models.ts

```diff
@@ -346,3 +346,24 @@ export interface RateLimitBucket {
   // last_take is a bigint in Postgres, which node-postgres turns into a string
   last_take: string;
 }
+
+export interface GenericIssuancePipelineDB {
```

Comment:
nit but the `DB` suffix makes me think this is a database rather than something comes from a database which i think is what you intend. worth renaming to something a little less confusing.

Reply:
Changed the suffix to 'Row'.